frontend/src/index.html

```diff
@@ -106,9 +106,9 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
                 </svg>
               </button>
             </div>
-            <div class="col-span-5 content-center pt-1 md:col-span-7">
+            <div class="mr-2 mt-1 content-center overflow-hidden">
```

Comment:
- Vertical gap can be better and more consistently implemented as a padding of the parent element.
- On the right, it's better to be a padding.

Reply:
updated ...